docs/src/examples/components/RadioGroup/Item/RadioGroupItemExample.shorthand.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { RadioGroup } from '@stardust-ui/react'
+
+const handleChange = () => {
+  alert('The radio checked value was changed!')
+}
+
+const RadioGroupItemExample = () => (
+  <RadioGroup.Item label="Make your choice" onChange={handleChange} />
+)
```

Comment:
If the RadioGroupItem should not be defined without the RadioGroup, I would suggest even in this examples, that it should be wrapped with RadioGroup, because the user is most likely to copy-paste the example without reasoning that it should be inside RadioGruop.

Reply:
Agree, the examples should be like for the Menu. Only RadioGroup together RadioGroupItem, without RadioGroupItem separately